json/stepperMotor.json

```diff
@@ -1,52 +1,57 @@
 {
+  "meta": {
+    "scriptGit": "https://gist.github.com/cb9e4207e37668c776eaae643701f92c.git",
+    "actuator": "true",
+    "scriptFile": "nemaStepper.groovy"
+  },
   "GenericNEMA11": {
-    "flangeDiameter": 21.0,
+    "mountScrew": "M3",
     "flangeHeight": 2.0,
-    "length": 27.0,
```

Comment:
Id need to write a custom serializer to get it alphabetical. Maps are treated as unsorted data and stored based on some internal algorithm. This one is hard...